layouts/default.vue

```diff
@@ -405,6 +345,33 @@ const auth = await useAuth()
 const user = await useUser()
 </script>
 <script>
+const NAV_ROUTES = [
+  {
+    label: 'Mods',
+    href: '/mods',
+  },
+  {
+    label: 'Plugins',
+    href: '/plugins',
+  },
+  {
+    label: 'Data Packs',
+    href: '/datapacks',
+  },
+  {
+    label: 'Shaders',
+    href: '/shaders',
+  },
+  {
+    label: 'Resource Packs',
+    href: '/resourcepacks',
+  },
+  {
+    label: 'Modpacks',
+    href: '/modpacks',
+  },
+]
+const SEARCH_PAGES = ['/mods', '/plugins', '/datapacks', '/shaders', '/resourcepacks', '/modpacks']
```

Comment:
No need to duplicate routes.

Reply:
Oops, I totally missed that! I'll fix it in the morning.